FogCreek/Glitch-Community
# Woolen crib: use micro-CMS for editing home page

## Links
* Remix link: http://woolen-crib.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328689/

## GIF/Screenshots:
![Screen Shot 2019-06-21 at 2 55 45 PM](https://user-images.githubusercontent.com/938722/59945150-b0630380-9434-11e9-8e1f-5800ba79f5d6.png)

## Changes:
* replace (some) /curated files with cached data from CMS
* add endpoints for reading & writing this cached data
* add homepage preview route

## How To Test:
* log into https://community-home-editor.glitch.me/
* update the content in the featured embed section (change it back when you're done 😃)
* log into http://woolen-crib.glitch.me/index/preview to see the changes (you will need to refresh after each change)
* Those changes will _not_ yet be visible on the "live" home page 
* Click "publish" in the banner at the top of the preview page
* You should be redirected to the home page, with your updates present

## Things to do _before_ deploying
* test the homepage update workflow with Ezra & Yasi

## Things to do _after_ deploying
* uncomment the lines in merge.sh, so that changes will continue to be synchronized from community to community-staging
